core/src/main/java/io/seata/core/rpc/netty/AbstractNettyRemotingServer.java

```diff
@@ -23,96 +23,82 @@
 import io.netty.handler.timeout.IdleState;
 import io.netty.handler.timeout.IdleStateEvent;
 import io.seata.common.util.NetUtil;
+import io.seata.core.protocol.HeartbeatMessage;
+import io.seata.core.protocol.ProtocolConstants;
 import io.seata.core.protocol.RpcMessage;
-import io.seata.core.rpc.ChannelManager;
 import io.seata.core.rpc.RemotingServer;
 import io.seata.core.rpc.RpcContext;
-import io.seata.core.rpc.TransactionMessageHandler;
 import io.seata.core.rpc.processor.Pair;
 import io.seata.core.rpc.processor.RemotingProcessor;
 import org.slf4j.Logger;
 import org.slf4j.LoggerFactory;
 
 import java.util.concurrent.ExecutorService;
 import java.util.concurrent.ThreadPoolExecutor;
+import java.util.concurrent.TimeoutException;
 
 /**
- * The type Rpc remoting server.
+ * The type abstract remoting server.
  *
- * @author slievrly
- * @author xingfudeshi@gmail.com
  * @author zhangchenghui.dev@gmail.com
+ * @since 1.3.0
  */
-public abstract class AbstractRpcRemotingServer extends AbstractRpcRemoting implements RemotingServer {
+public abstract class AbstractNettyRemotingServer extends AbstractNettyRemoting implements RemotingServer {
 
-    private static final Logger LOGGER = LoggerFactory.getLogger(AbstractRpcRemotingServer.class);
+    private static final Logger LOGGER = LoggerFactory.getLogger(AbstractNettyRemotingServer.class);
 
-    private final RpcServerBootstrap serverBootstrap;
-
-    private TransactionMessageHandler transactionMessageHandler;
-
-    /**
-     * Instantiates a new Rpc remoting server.
-     *
-     * @param messageExecutor   the message executor
-     * @param nettyServerConfig the netty server config
-     */
-    public AbstractRpcRemotingServer(final ThreadPoolExecutor messageExecutor, NettyServerConfig nettyServerConfig) {
+    public AbstractNettyRemotingServer(ThreadPoolExecutor messageExecutor) {
         super(messageExecutor);
-        serverBootstrap = new RpcServerBootstrap(nettyServerConfig);
```

Comment:
vs  AbstractNettyRemotingClient, why remove Bootstrap?

Reply:
Specific Bootstrap has subclasses to implement, for example, now put bootstarp in NettyRemotingServer.